test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1,34 +1,37 @@
-import { StructureDefinitionExporter, Package } from '../../src/export';
-import { FSHTank, FSHDocument } from '../../src/import';
+import { Package, StructureDefinitionExporter } from '../../src/export';
```

Comment:
The diff displayed in GitHub is very messed up for this file.  Can I assume that the only substantial changes you made were to add the tests around logic/resource, and everything else was just moved around?

Reply:
Correct. I added `describe` for "#Parent", "#LogicalModel", "#Resource". and "#AddElementRule". I can't recall off the top of my head, but there might have been several minor fixes elsewhere.